ui/src/components/Engine.jsx

```diff
@@ -217,6 +218,17 @@ class Engine extends React.Component {
       this.setState({
         consoleOpen: !this.state.consoleOpen,
       });
+      if(this.state.consoleOpen){
+        this.resizable.updateSize({ height: 0 });
```

Comment:
Why is the `height` state doubled? (on the component and locally)

Ideally there is a single source of truth, not something that needs to be synced.

Reply:
8f33bc7